tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -478,64 +525,66 @@ def test_44804(self):
             LOGGER.info("List %s-obj2 to dest-bucket1", obj_type)
-            self.objects_in_buckets(bucket="src-bucket1", objects=[f"{obj_type}-obj1"])
-            self.objects_in_buckets(bucket="dest-bucket1", objects=[f"{obj_type}-obj2"])
-            self.parallel_copy_object(("src-bucket1", f"{obj_type}-obj1", "dest-bucket1",
-                                      f"{obj_type}-obj2"),
-                                      ("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
-                                      f"{obj_type}-obj1"))
-            resp = self.s3_obj.object_info("dest-bucket1", f"{obj_type}-obj2")
+            objects_in_buckets(bucket=self.des_bkt1, objects=[f"{obj_type}-obj2"],
+                               s3_test_obj=self.s3_obj)
+            self.parallel_copy_object((self.src_bkt1, f"{obj_type}-obj1", self.des_bkt1,
+                                      f"{obj_type}-obj2"), (self.des_bkt1, f"{obj_type}-obj2",
+                                      self.src_bkt1, f"{obj_type}-obj1"))
+            resp = self.s3_obj.object_info(self.des_bkt1, f"{obj_type}-obj2")
             assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
-            self.parallel_copy_object(("src-bucket1", f"{obj_type}-obj1", "dest-bucket1",
-                                      f"{obj_type}-obj2"),
-                                      ("dest-bucket1", f"{obj_type}-obj3", "src-bucket1",
-                                      f"{obj_type}-obj4"))
-            self.validate_copy_content("src-bucket1", f"{obj_type}-obj1",
-                                       "dest-bucket1", f"{obj_type}-obj2")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj3",
-                                       "src-bucket1", f"{obj_type}-obj4")
-            self.multi_parallel_copy_object(("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
-                                            f"{obj_type}-obj3"),
-                                            ("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
-                                            f"{obj_type}-obj4"),
-                                            ("dest-bucket1", f"{obj_type}-obj2", "dest-bucket2",
-                                            f"{obj_type}-obj4"),
-                                            numproc=3)
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "src-bucket1", f"{obj_type}-obj3")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "src-bucket1", f"{obj_type}-obj4")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "dest-bucket2", f"{obj_type}-obj4")
+            self.parallel_copy_object((self.src_bkt1, f"{obj_type}-obj1", self.des_bkt1,
+                                      f"{obj_type}-obj2"), (self.des_bkt1, f"{obj_type}-obj3",
```

Comment:
Line 557 : We can move the second tuple being passed to a new line here for readability like how its done in line no. 293 .
Also, we can indent the arguments passed inside the 1st tuple like how its done in line no. 292 .
Applicable to other calls to parallel_copy_object as well.

Reply:
readability is understood. We also take care of reducing no. of lines of code and following the coding standards. line 293 and 292 anyways would need 3 lines , hence kept like that. line 557 needed 4 lines if not added on same line as tuple1.